packages/ember-views/tests/views/view/view_lifecycle_test.js

```diff
@@ -1,243 +0,0 @@
-import { context } from 'ember-environment';
-import run from 'ember-metal/run_loop';
-import jQuery from 'ember-views/system/jquery';
-import EmberView from 'ember-views/views/view';
-import { compile } from 'ember-template-compiler';
-import { registerHelper } from 'ember-htmlbars/helpers';
-
-const originalLookup = context.lookup;
-let lookup, view;
-
-import { test, testModule } from 'internal-test-helpers/tests/skip-if-glimmer';
-
-QUnit.module('views/view/view_lifecycle_test - pre-render', {
```

Comment:
@krisselden @rwjblue @wycats Need confirmation that this thing can just go away. From What I can tell it's testing cases that do not make sense in the post-view era.